telcell/data/models.py

```diff
@@ -21,6 +21,10 @@ class Measurement:
     timestamp: datetime
     extra: Mapping[str, Any]
 
+    @property
+    def latlon(self) -> Tuple[float, float]:
```

Comment:
We might be able to avoid the eternal debate "latlon" vs "latlong" by naming this `location`?